lib/dl_formula_ref/dl_formula_ref/categories/window.py

```diff
@@ -80,34 +83,27 @@
                     "calculating a measure}. These dimensions define how values are split into "
                     "groups and therefore have fixed values in each group.\n"
                     "\n"
-                    "If you specify a dimension that was not used to build the chart, it won't "
+                    "   If you specify a dimension that was not used to build the chart, it won't "
                     "have a fixed value and the value can be different in each group row. As a "
                     "result, it will be impossible to determine which value of this dimension "
                     "must be used to calculate the measure. This limitation applies to the "
                     "`WITHIN` and `AMONG` grouping types.\n"
                     "\n"
-                    "    Examples:\n"
-                    "    - Valid: `RANK(SUM([Profit]) WITHIN [Category])` in the chart with "
-                    "grouping by the `[Order Date]` and `[Category]` dimensions.\n"
-                    "    - Allowed: `RANK(SUM([Profit]) WITHIN [City])` in the chart with "
-                    "grouping by the `[Category]` dimension,       `[City]` does not participate "
-                    "in the grouping.\n"
-                    "    - Not valid: `RANK(SUM([Profit]) WITHIN [Category])` in a chart with "
-                    "grouping by the `[Order Date]` and `[City]` dimensions.\n"
-                    "    - Not valid: `RANK(SUM([Profit]) AMONG [City])` in a chart with grouping "
-                    "by the `[Order Date]` and `[Category]` dimensions.\n"
-                    "1. The **Filters** section doesn't affect the chart grouping, so if the "
-                    "dimension is only in this chart section, you can't use it in the window "
-                    "function.\n"
-                    "\n"
-                    "    Example:\n"
-                    "    - Chart type: **Table**.\n"
-                    "    - In the **Columns** section, the `Category` dimension and the "
-                    "`SUM(SUM([Sales] BEFORE FILTER BY [Date])` expression are added.\n"
-                    "    - The `Date` dimension is added to the **Filters** section.\n"
-                    "\n"
-                    "    This will result in an error because the `Date` dimension isn't used to "
-                    "build the chart.\n"
+                    "   Examples:\n"
+                    "   * Correct formula: `RANK(SUM([Profit]) WITHIN [Category])` in the chart "
+                    "with grouping by the `[Order Date]` and `[Category]` dimensions. "
+                    "`[Category]` is used when building the chart.\n"
+                    "   * Valid formula: `RANK(SUM([Profit]) WITHIN [Category])` in the chart with "
```

Comment:
Probably should be `Correct formula` for consistency?

Reply:
fixes